packages/dashboards/examples/src/components/docs/tutorials/customization/customization.module.ts

```diff
@@ -35,5 +36,6 @@ const routes: Routes = [
     imports: [
         RouterModule.forChild(routes),
     ],
+    providers: [ConfiguratorHeadingService],
```

Comment:
Do you mind adding a step to the tutorial letting the user know they need to do this?

A new list item here would be good:
https://github.com/solarwinds/nova/blob/9a15a6d9e6d39b3e4cc5d0dc8ecf1422338c09b4/packages/dashboards/examples/src/components/docs/tutorials/customization/data-source-configurator/custom-data-source-configurator-docs.component.html#L67

Reply:
Just added it. Can you check the wording, please?

Reply:
![image](https://user-images.githubusercontent.com/22597043/120515367-eb1e9600-c3d6-11eb-8bef-df1a6249f1c8.png)

Reply:
I like it! See this comment here as well. I should have mentioned it, but I got interrupted by something: https://github.com/solarwinds/nova/pull/313/files#r644216308